test/Microsoft.ML.StaticPipelineTesting/Training.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Data.StaticPipe;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Calibration;
+using Microsoft.ML.Runtime.Learners;
+using System;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.StaticPipelineTesting
+{
+    public sealed class Training : MakeConsoleWork
```

Comment:
>MakeConsoleWork [](start = 35, length = 15)

can we rename this to something that indicates that it's a base class for tests?
Like `BaseTestClassWithConsole` ?